src/components/entry/Entry.jsx

```diff
@@ -11,6 +19,36 @@ export default function Entry({ entry }) {
 				lng: {entry.location[0]} <br />
 				lat: {entry.location[1]}
 			</Typography>
+			<RenderIf isTrue={!isDeleteMode}>
+				<Button
+					type="button"
+					variant="smallDo"
+					onClick={() => {
+						toggleMode(isDeleteMode, setIsDeleteMode);
+					}}
+				>
+					X
+				</Button>
+			</RenderIf>
```

Comment:
Using a separated component to implement a simple condition is highly over-engineered. This solution is way more complex than it needs to be. The additional component adds more code and forces you to switch contexts when reading the code, which makes it hart to understand.


```suggestion
			{!isDeleteMode &&  
			<Button
					type="button"
					variant="smallDo"
					onClick={() => {
						toggleMode(isDeleteMode, setIsDeleteMode);
					}}
				>
					X
			</Button>}
```

Reply:
Hey Dominik, thanks for your review. I was struggling with my edit function over the weekend and just couldn't get it to work. In the process, the idea with renderif came up. I will rebuild that, because it is actually over without edit. Also the toggle function is so originated, because I needed the same function in several components. In all the frustration I have after deleting all the edit things, no longer get the brain node solved for cleaning up. I am quickly rebuilding... Thanks!